orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/AmazonImageFinder.java

```diff
@@ -85,6 +91,11 @@ public boolean test(AmazonImage amazonImage) {
             .sorted()
             .collect(Collectors.toList());
 
+    if (allMatchedImages.size() == MAX_SEARCH_RESULTS) {
```

Comment:
```suggestion
    if (allMatchedImages.size() >= MAX_SEARCH_RESULTS) {
```

Typically speaking, I'm pretty pessimistic of equality operators in these cases not introducing a regression in months, years to come.